content/pt-br/container.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Contêineres
+status: Completed
+category: tecnologia
+---
+
+## O que é
+Um contêiner é um processo em execução com restrições de recursos e capacidade gerenciadas pelo sistema operacional. Os arquivos disponíveis para o processo de contêiner são empacotados como uma imagem de contêiner. Os contêineres são executados um ao lado do outro na mesma máquina, mas normalmente o sistema operacional impede que os processos de contêiner separados interajam entre si.
+
+## Problema relacionado
+Antes da existência dos contêineres, eram necessárias máquinas separadas eram para executar os aplicativos. Cada máquina exigiria seu próprio sistema operacional, que consome CPU, memória e espaço em disco, tudo para que um aplicativo individual funcione. Além disso, a manutenção, atualização e inicialização de um sistema operacional é outra fonte significativa de trabalho.
```

Comment:
I think here we could specify `TOIL` instead of `trabalho` (since TOIL means repetitive work) or use instead `trabalho repetitivo`